iphone/CoreApi/CoreApi/Common/MWMGeoUtil.mm

```diff
@@ -31,10 +32,8 @@ @implementation Measure
 */
 
 - (NSString*) valueAsString {
-  if (self.value > 9.999)
-    return [NSString stringWithFormat:@"%.0f", self.value];
-  else
-    return [NSString stringWithFormat:@"%.1f", self.value];
+  auto outString = measurement_utils::ToStringPrecision(self.value, self.value >= 10.0 ? 0 : 1);
```

Comment:
```suggestion
  auto const outString = measurement_utils::ToStringPrecision(self.value, self.value >= 10.0 ? 0 : 1);
```

Reply:
Noted! Commit will be amended accordingly...